### PR TITLE
Add new committee members after the early 2022 EGMs

### PR DESCRIPTION
**Pull request by Danyc0**

### CODE REVIEW

File: _data/committee/2021-22.yaml [hunk 1]
```diff
@@ -115,40 +139,53 @@ people:
...
-  - name: Victoria Tilley
-    pronouns: She/Her, They/Them
-    discordtag: Sapphyre#5656
-    role: Equality, Diversity, and Inclusivity Officer
-    picture: /assets/committee/2021-22/mini/victoria.jpg
-    start-date: 1/7/2021
-    end-date: 28/1/2022
-    bio: >
-      Heya! I'm Victoria and this year I'll be making sure that everyone feels included and represented at all of our events and in our School. 
-      Outside of this, I enjoy game development, airsoft, snowboarding and playing guitar. Come to me if you need anything!
```
Comment (LikkanChung) [inline _data/committee/2021-22.yaml:127]: I think keeping both is what @AlexJBanks wanted

Comment (LikkanChung) [inline _data/committee/2021-22.yaml:127]: I think keeping both is what @AlexJBanks wanted

Comment (Danyc0) [reply _data/committee/2021-22.yaml:127]: ooh yea, will fix

Comment (Danyc0) [reply _data/committee/2021-22.yaml]: Fixed

File: _data/committee/2021-22.yaml [hunk 2]
```diff
@@ -60,6 +71,18 @@ people:
       I’m hoping to make sure that all first year students settle into our school as comfortably as possible and give them many opportunities to socialise, 
       make friends and learn more about what computer science has to offer 💜.
 
+  - name: Gokul Menon
+    pronouns: He/Him
+    discordtag: laterflow#8193
+    role: First Year Rep
+    start-date: 29/1/2022
```
Comment (JTWWilson) [inline _data/committee/2021-22.yaml:78]: Gokul started 20/10/2021

File: _data/committee/2021-22.yaml [hunk 3]
```diff
@@ -115,6 +138,18 @@ people:
       Hello, I'm Ryan and I'm going into my 2nd year studying Computer Science. I will be responsible for all of your sports and e-Sports events this year -- 
       please let me know what you want to see! When I'm not sleeping I'm either speedrunning, participating in hackathons or working on programming projects.
 
+  - name: Aarogya Yadav
+    pronouns: He/Him
+    discordtag: Arcane34#4178
+    role: Sports Secretary
+    picture: /assets/committee/2021-22/mini/aarogya.jpg
+    start-date: 29/1/2022
```
Comment (JTWWilson) [inline _data/committee/2021-22.yaml:146]: Aarogya, Tori and Matty started on 15/02/2022

**Review by JTWWilson — CHANGES_REQUESTED** [review]

File: _data/committee/2021-22.yaml (hunk 2 above)
Comment (Danyc0) [reply _data/committee/2021-22.yaml]: Fixed

File: _data/committee/2021-22.yaml (hunk 3 above)
Comment (Danyc0) [reply _data/committee/2021-22.yaml]: Fixed

**Review by JTWWilson — APPROVED** [review]
Looks good, think this is the best way to have Victoria properly represented

**Review by LikkanChung — APPROVED** [review]

**Comment by Danyc0** [comment]
> Looks good, think this is the best way to have Victoria properly represented

Yea it was just me not thinking, I had meant to do it this way anyway

**Pull request merged**